### PR TITLE
Use Ruby 2.3 and Rubocop 0.55

### PR DESCRIPTION
**Pull request by pathawks**

### CODE REVIEW

File: lib/jekyll/page_without_a_file.rb [hunk 1]
```diff
@@ -2,8 +2,10 @@
 
 module Jekyll
   class PageWithoutAFile < Page
+    # rubocop:disable Naming/MemoizedInstanceVariableName
     def read_yaml(*)
```
Comment (pathawks) [inline lib/jekyll/page_without_a_file.rb]: Rubocop wants to call this method `@read_yaml`

Comment (DirtyF) [reply lib/jekyll/page_without_a_file.rb]: The method is already called `read_yaml`, Rubocop wants us to name the `@data` instance `@read_yaml` no?

Comment (pathawks) [reply lib/jekyll/page_without_a_file.rb]: Oh! Yeah, ok, that makes a _lot_ more sense 🤦‍♂️

**Comment by DirtyF** [comment]
@jekyllbot: merge +dev

**Comment by pathawks** [comment]
Failing on Travis because Ruby 2.2 and GitHub Pages. 😿

**Pull request merged**